[FR] Add `osu!mapping` (#5698)
ppy/osu-wiki: wiki/osu!mapping/fr.md

---
tags:
  - interview
  - pishi
  - pishifat
  - tutorial
  - video
  - videos
  - YouTube

> ---

Reviewer: Avant la ligne 10, ajoute ceci : 

`- tutoriel`
`- vidéos`

Reviewer: J'aurais du préciser de les ajouter, **sans retirer ceux déjà présent.**

Author: désolé, c'était pourtant plus intelligent

Author: y'a un bouton commit all changes proposed ou je vais edit direct au lieu de commit changes a chaque request

# osu!mapping

> *A ne pas être confondue avec [Beatmapping](/wiki/Beatmapping).*

Reviewer: ```suggestion
*A ne pas confondre avec [Beatmapping](/wiki/Beatmapping).*
```

> **osu!mapping** est une série de vidéo hebdomadaire creé par ![](/wiki/shared/flag/US.gif) **[pishifat](https://osu.ppy.sh/users/3178418)** a propos de l'écosystème du [mapping](/wiki/Beatmapping) et [modding](/wiki/Modding). Ces vidéos peuvent être retrouvées sur [la chaîne youtube de pishifat](https://www.youtube.com/user/Pishifat), la grande majorité des gens les connaissent sous le nom de "pishifat videos" ou "pishi videos".

Reviewer: ```suggestion
**osu!mapping** est une série de vidéo hebdomadaire créé par ![](/wiki/shared/flag/US.gif) **[pishifat](https://osu.ppy.sh/users/3178418)** a propos de l'écosystème du [mapping](/wiki/Beatmapping) et du [modding](/wiki/Modding). Ces vidéos peuvent être retrouvées sur [la chaîne youtube de pishifat](https://www.youtube.com/user/Pishifat). La grande majorité des gens connais ces vidéos sous le nom de "pishifat videos" ou "pishi videos".
```

> La série est devenue tellement bien réputée dans la communauté osu! que c'est une ressource qui est généralement recommandé au nouveaux mappers.[L'équipe d'osu!](/wiki/People/The_Team) a récompensé pishifat du badge[Contributeur de la communauté](/wiki/People/Community_Contributors) pour sa "performance exemplaire dans la construction et l'éducation autour du domaine des beatmap" et pour remercier tout ses efforts mit dans ce projet.

Reviewer: ```suggestion
La série est devenue tellement réputée dans la communauté osu! que c'est une ressource qui est généralement recommandé aux nouveaux mappeurs. [L'équipe d'osu!](/wiki/People/The_Team) à récompensé pishifat du badge [community contributors](/wiki/People/Community_Contributors) pour sa "performance exemplaire dans la construction et l'éducation autour du domaine du mapping" et pour remercier tout ses efforts investi dans ce projet.
```

> La suite de vidéo hebdomadaire osu!mapping s'arrêta en Janvier 2019, mais il y a toujours des vidéos prévus qui compte être ajoutés dans la série.

Reviewer: ```suggestion
La suite des vidéos hebdomadaire de l'osu!mapping s'arrêta en janvier 2019, mais il y a toujours des vidéos prévus qui compte être ajoutés dans la série.
```

## Liens

> - **[chaîne youtubel](https://www.youtube.com/user/Pishifat)**

Reviewer: ```suggestion
- **[Chaîne YouTube](https://www.youtube.com/user/Pishifat)**
```

> - [Patreon](https://www.patreon.com/pishifat) (currently closed)

Reviewer: ```suggestion
- [Patreon](https://www.patreon.com/pishifat) (actuellement fermé)
```

- [Twitter](https://twitter.com/pishifat)

> ## Histoires

Reviewer: ```suggestion
## Histoire
```

> osu!mapping a débuté comme une réponse a un [post reddit](https://www.reddit.com/r/osugame/comments/4upysl/request_for_mappers_of_this_sub/) demandant au mappers de créer des vidéos commentés sur le mapping. Le 31 juillet 2016, pishifat posta sa première vidéo: [osu!mapping: concepts](https://www.youtube.com/watch?v=oL3az8FhlB4). Dans la vidéo, il explique comment il va faire ces vidéos comme ont lui a demandé sur reddit, mais il finit par décider de faire autrement car un commentaire direct de mapping serait indigeste et n'expliquerait donc pas bien aux nouveaux mappers l'immense variété des concept de mapping. A la place, il a choisi de commencer une série de vidéos ou chaque épisode se concentrerai sur un concept spécifique, expliquant son utilisation et importance dans sa globalité sans laisser d'omis.

Reviewer: ```suggestion
l'osu!mapping a débuté comme une réponse a un [post reddit](https://www.reddit.com/r/osugame/comments/4upysl/request_for_mappers_of_this_sub/) demandant au mappeurs de créer des vidéos commentés sur le mapping. Le 31 juillet 2016, pishifat posta sa première vidéo : [osu!mapping: concepts](https://www.youtube.com/watch?v=oL3az8FhlB4). Dans la vidéo, il explique comment il va faire ces vidéos comme cela à été demander sur reddit, mais il finit par décider de faire autrement car un commentaire direct de mapping serait indigeste et n'expliquerait donc pas bien aux nouveaux mappeurs l'immense variété des concept de mapping. A la place, il a choisi de commencer une série de vidéos ou chaque épisode se concentrerais sur un concept spécifique, expliquant son utilisation et importance dans sa globalité sans laisser de floues.
```

<!-- TODO: this is not really concrete history -->

> La série c'est concentré  au debut sur le coté tutoriel, mais a commendé a se diversifier après quelques mois pour inclure des discussions a propos de l'état du mapping et du modding, commentaires personnels/opinions, et interviews. Il ya a aussi eu des projets de recherches présentés sous la forme de vidéos d'osu!mapping, nottament [une étude des influences des mappers](https://www.youtube.com/watch?v=TigwDc5iOcw).

Reviewer: ```suggestion
La série s'est concentré  au début sur le coté tutoriel, mais a commencé a se diversifier après quelques mois pour inclure des discussions a propos de l'état du mapping et du modding, des commentaires personnels et des opinions, et des interviews. Il y a a aussi eu des projets de recherches présentés sous la forme de vidéos d'osu!mapping, notamment [une étude des influences des mappeurs](https://www.youtube.com/watch?v=TigwDc5iOcw).
```

## Episodes

> 

Reviewer: A partir d'ici, vaut mieux garder le titre originel des vidéos. Je suis sympa je vais te faire les suggestions qu'il faut pour corriger cela, comme c'est la première fois que tu contribue. Mais la prochaine fois, je dirait de garder les titres comme ça en anglais.

Author: ok

> 1. [notions](https://www.youtube.com/watch?v=oL3az8FhlB4)

Reviewer: ```suggestion
1. [concepts](https://www.youtube.com/watch?v=oL3az8FhlB4)
```

> 2. [accentuation des espacements](https://www.youtube.com/watch?v=rAYm50d6XMw)

Reviewer: ```suggestion
2. [spacing emphasis](https://www.youtube.com/watch?v=rAYm50d6XMw)
```

> 3. [espacement de base](https://www.youtube.com/watch?v=v1_HRo-lWWI)

Reviewer: ```suggestion
3. [basic spacing](https://www.youtube.com/watch?v=v1_HRo-lWWI)
```

> 4. [rythme de base](https://www.youtube.com/watch?v=3BC_I2ufqC8)

Reviewer: ```suggestion
4. [basic rhythm](https://www.youtube.com/watch?v=3BC_I2ufqC8)
```

> 5. [rythme irrégulier](https://www.youtube.com/watch?v=c7jYwSVjiSo)

Reviewer: ```suggestion
5. [irregular rhythm](https://www.youtube.com/watch?v=c7jYwSVjiSo)
```

> 6. [l'éditeur](https://www.youtube.com/watch?v=2k8s0gBg9yU)

Reviewer: ```suggestion
6. [the editor](https://www.youtube.com/watch?v=2k8s0gBg9yU)
```

> 7. [esthétique](https://www.youtube.com/watch?v=FCB5rLXqWKY)

Reviewer: ```suggestion
7. [aesthetics](https://www.youtube.com/watch?v=FCB5rLXqWKY)
```

8. [1 3 2 4](https://www.youtube.com/watch?v=qmCEfh865b4)
9. [flow + circles](https://www.youtube.com/watch?v=ig7nTSbWwVc)
10. [hitsounds](https://www.youtube.com/watch?v=Lz6D7HPJ0eQ)
11. [flow + sliders](https://www.youtube.com/watch?v=GYGatwQszwQ)

> 12. [trucs](https://www.youtube.com/watch?v=nk8h1HVb8_A)

Reviewer: ```suggestion
12. [gimmicks](https://www.youtube.com/watch?v=nk8h1HVb8_A)
```

13. ["loved"](https://www.youtube.com/watch?v=CWodsLSmNeE)
14. [overmapping](https://www.youtube.com/watch?v=lORCxluEv3g)

> 15. [amusant vs. bien](https://www.youtube.com/watch?v=C4v1QVJozGs)

Reviewer: ```suggestion
15. [fun vs. good](https://www.youtube.com/watch?v=C4v1QVJozGs)
```

16. [nouveaux combos](https://www.youtube.com/watch?v=neycvFQ5Ug4)
17. [progression](https://www.youtube.com/watch?v=VBPYemKlTic)
18. [spread](https://www.youtube.com/watch?v=7cFzwRIxte4)
19. [speed-ranking](https://www.youtube.com/watch?v=VZEQNoxTap0)
20. [modding](https://www.youtube.com/watch?v=jhsBN3nolw0)
21. [compétence des joueurs](https://www.youtube.com/watch?v=rkU082YUOPY)
22. [thèmes](https://www.youtube.com/watch?v=WfrQrQCA3Vg)
23. [filtering](https://www.youtube.com/watch?v=tchb7Gm1_lY)
24. [streams](https://www.youtube.com/watch?v=oJZ0_MrdSFM)

> 25. [critique de base](https://www.youtube.com/watch?v=lh19LmL7Nvw)

Reviewer: ```suggestion
25. [core criticism](https://www.youtube.com/watch?v=lh19LmL7Nvw)
```

26. [timing](https://www.youtube.com/watch?v=xauZuMPgiQw)

> 27. [sondage: meilleure de 2016](https://www.youtube.com/watch?v=jdpTmlvW6ow)

Reviewer: ```suggestion
27. [best of 2016 poll](https://www.youtube.com/watch?v=jdpTmlvW6ow)
```

> 28. [perspectives](https://www.youtube.com/watch?v=ZXakpTkXEC0)

Reviewer: ```suggestion
28. [perspectives](https://www.youtube.com/watch?v=ZXakpTkXEC0)
```

> 29. [plaintes](https://www.youtube.com/watch?v=5amKzp3IQRU)

Reviewer: ```suggestion
29. [complaints](https://www.youtube.com/watch?v=5amKzp3IQRU)
```

30. [difficulty spikes](https://www.youtube.com/watch?v=lkpfDgG_oxE)

> 31. [mots a la mode](https://www.youtube.com/watch?v=bdaZ22it5Tw)

Reviewer: ```suggestion
31. [buzzwords](https://www.youtube.com/watch?v=bdaZ22it5Tw)
```

32. [jumps](https://www.youtube.com/watch?v=-naduAbrBnc)

> 33. [critères de classement standard](https://www.youtube.com/watch?v=QEJBY3uwBTg)

Reviewer: ```suggestion
33. [standard ranking criteria](https://www.youtube.com/watch?v=QEJBY3uwBTg)
```

34. [aspire](https://www.youtube.com/watch?v=f1xj0P9XOow)
35. [choix de la chanson](https://www.youtube.com/watch?v=WlXKFagzo_o)
36. [finding mods](https://www.youtube.com/watch?v=tCydtxk8yKA)
37. [filler rhythm](https://www.youtube.com/watch?v=87CX7bxov7o)
38. [mapping style](https://www.youtube.com/watch?v=UhOfPakTtgM)
39. [qualified maps](https://www.youtube.com/watch?v=TqYqNf5m-aY)
40. [variety](https://www.youtube.com/watch?v=Z7IY0gDwSQo)

> 41. [efficacité de l'éditeur](https://www.youtube.com/watch?v=VD0HwSErWjw)

Reviewer: ```suggestion
41. [editor efficiency](https://www.youtube.com/watch?v=VD0HwSErWjw)
```

42. [end of the BAT](https://www.youtube.com/watch?v=ZuL5LfRZDJM)
43. [view options](https://www.youtube.com/watch?v=VawvhkNZl4Y)
44. [retour honnête](https://www.youtube.com/watch?v=5xC2oY1Fb70)

> 45. [lignes vertes](https://www.youtube.com/watch?v=J4OKPHZzJ1k)

Reviewer: ```suggestion
45. [green lines](https://www.youtube.com/watch?v=J4OKPHZzJ1k)
```

> 46. [collaborations](https://www.youtube.com/watch?v=Ly0u6n2qrOE)

Reviewer: ```suggestion
46. [collabs](https://www.youtube.com/watch?v=Ly0u6n2qrOE)
```

> 47. [décisions des nouveaux mappers](https://www.youtube.com/watch?v=7WSjFFyzGAk)

Reviewer: ```suggestion
47. [new mapper decisions](https://www.youtube.com/watch?v=7WSjFFyzGAk)
```

> 48. [réputation](https://www.youtube.com/watch?v=y3kkmH8UQRI)

Reviewer: ```suggestion
48. [reputation](https://www.youtube.com/watch?v=y3kkmH8UQRI)
```

49. [hp drain](https://www.youtube.com/watch?v=1h9fJ31WOMc)
50. ["wub" maps](https://www.youtube.com/watch?v=eoX3-2HYIsA)
51. [disqualification](https://www.youtube.com/watch?v=3PmlbKa5VjA)

> 52. [flux de maps](https://www.youtube.com/watch?v=l_0BC5kmXeI)

Reviewer: ```suggestion
52. [workflow](https://www.youtube.com/watch?v=l_0BC5kmXeI)
```

> 53. [une année](https://www.youtube.com/watch?v=c0bPx5L1vO0)

Reviewer: ```suggestion
53. [one year](https://www.youtube.com/watch?v=c0bPx5L1vO0)
```

> 54. [ancien éditeurs](https://www.youtube.com/watch?v=glvvgl21txc)

Reviewer: ```suggestion
54. [old editors](https://www.youtube.com/watch?v=glvvgl21txc)
```

55. [alternating maps](https://www.youtube.com/watch?v=Lo8RPVjsrto)
56. [approval](https://www.youtube.com/watch?v=U6aNt8z2OY0)
57. [circle size](https://www.youtube.com/watch?v=AZuWQKAZ83E)
58. [moddingv2](https://www.youtube.com/watch?v=gZxsB98HGq0)
59. [alien](https://www.youtube.com/watch?v=Lj0SSyjtI6A)

> 60. [nostalgie](https://www.youtube.com/watch?v=hjcoIQhX_KI)

Reviewer: ```suggestion
60. [nostalgia](https://www.youtube.com/watch?v=hjcoIQhX_KI)
```

61. [difficultés basses](https://www.youtube.com/watch?v=LEBlmgfQfC0)

> 62. [copier d'autres mappers](https://www.youtube.com/watch?v=YjocANVGp2I)

Reviewer: ```suggestion
62. [copying other mappers](https://www.youtube.com/watch?v=YjocANVGp2I)
```

63. [difficulty inflation](https://www.youtube.com/watch?v=Gewm_IQm3Bk)
64. [limitation maps](https://www.youtube.com/watch?v=9ayIcyqWhqI)

> 65. ["loved" -- a year later](https://www.youtube.com/watch?v=xMcW8VjkXvQ)

Reviewer: ```suggestion
65. [loved -- a year later](https://www.youtube.com/watch?v=xMcW8VjkXvQ)
```

66. [waveforms](https://www.youtube.com/watch?v=g-6uUT1vX8Q)

> 67. [processus de ranking](https://www.youtube.com/watch?v=9Bp2e2upFcQ)

Reviewer: ```suggestion
67. [ranking process](https://www.youtube.com/watch?v=9Bp2e2upFcQ)
```

68. [scorev2 + sliderhead leniency](https://www.youtube.com/watch?v=n4bUdiDzR8g)

> 69. [imperfections de l'éditeur](https://www.youtube.com/watch?v=Q30GusNb28w)

Reviewer: ```suggestion
69. [editor imperfections](https://www.youtube.com/watch?v=Q30GusNb28w)
```

70. [contests](https://www.youtube.com/watch?v=d-cbB_gFPk4)
71. [resources](https://www.youtube.com/watch?v=4FOGaKp3aTY)
72. [perfectly overlapping sliderbodies](https://www.youtube.com/watch?v=edotqxG-3So)

> 73. [variations sur lazer](https://www.youtube.com/watch?v=9S1N_F1bmEg)

Reviewer: ```suggestion
73. [lazer variations](https://www.youtube.com/watch?v=9S1N_F1bmEg)
```

74. [bn tests](https://www.youtube.com/watch?v=q3dIhF9Icas)
75. [slidershapes](https://www.youtube.com/watch?v=ecolEJxnbOc)

> 76. [raisons derriere la musique](https://www.youtube.com/watch?v=EO9yWFSSL3Q)

Reviewer: ```suggestion
76. [reasons behind music](https://www.youtube.com/watch?v=EO9yWFSSL3Q)
```

77. [nominator tiers](https://www.youtube.com/watch?v=BIusSGJSM3o)
78. [hip hop](https://www.youtube.com/watch?v=yQTsqNmKar8)
79. [the end of bubbles](https://www.youtube.com/watch?v=9Za-1_hxkxE)
80. [not favorites of 2017](https://www.youtube.com/watch?v=Yhsv25Iw1zg)

> 81. [deaf](https://www.youtube.com/watch?v=58R7mGGWCkA)

Reviewer: ```suggestion
81. [without audio](https://www.youtube.com/watch?v=58R7mGGWCkA)
```

> 82. [conseils d'éditeur](https://www.youtube.com/watch?v=npkllaA0A_U)

Reviewer: ```suggestion
82. [editor tips](https://www.youtube.com/watch?v=npkllaA0A_U)
```

> 83. [hw & creativity](https://www.youtube.com/watch?v=8JLJ9zBAQms)

Reviewer: ```suggestion
83. [hollow wings & creative maps](https://www.youtube.com/watch?v=8JLJ9zBAQms)
```

84. [a commentary](https://www.youtube.com/watch?v=sMXVCdQyp7w)
85. [slider velocity](https://www.youtube.com/watch?v=s9oDCdOaIIc)
86. [2b](https://www.youtube.com/watch?v=8teIBNQfr7Q)

> 87. [première map ranked](https://www.youtube.com/watch?v=El0PXv2em8A)

Reviewer: ```suggestion
87. [first ranked map](https://www.youtube.com/watch?v=El0PXv2em8A)
```

88. [metadata](https://www.youtube.com/watch?v=qjNPoTOyNL4)
89. [skins](https://www.youtube.com/watch?v=y3s241f32iU)

> 90. [discussions sur le modding](https://www.youtube.com/watch?v=0dwvDlU9SWQ)

Reviewer: ```suggestion
90. [modding discussions](https://www.youtube.com/watch?v=0dwvDlU9SWQ)
```

91. [input devices](https://www.youtube.com/watch?v=zm0pgxe-I0c)
92. [hanzer & miss you](https://www.youtube.com/watch?v=HFSNFWu5wws)
93. [hitsounding & rhythm choice](https://www.youtube.com/watch?v=-DoZnsMzaMw)

> 94. [en 5 minutes](https://www.youtube.com/watch?v=VZLlmaxUXto)

Reviewer: ```suggestion
94. [in 5 minutes](https://www.youtube.com/watch?v=VZLlmaxUXto)
```

> 95. [nouveautés mineur de l'éditeur](https://www.youtube.com/watch?v=TL-kIc_zNs0)

Reviewer: ```suggestion
95. [minor editor update -- may 2018](https://www.youtube.com/watch?v=TL-kIc_zNs0)
```

> 96. [RLC & following music](https://www.youtube.com/watch?v=8yh4btib9AQ)

Reviewer: ```suggestion
96. [RLC & "following music"](https://www.youtube.com/watch?v=8yh4btib9AQ)
```

97. [mappers](https://www.youtube.com/watch?v=TigwDc5iOcw)
98. [drum n bass](https://www.youtube.com/watch?v=Y9KnfQnssxc)
99. [circle/slider only](https://www.youtube.com/watch?v=xbRKUFLdcSM)
100. [playfield usage](https://www.youtube.com/watch?v=CVRULow2tJo)

> 101. [nouveaux mappers](https://www.youtube.com/watch?v=UjHVyBJudgg)

Reviewer: ```suggestion
101. [new mappers](https://www.youtube.com/watch?v=UjHVyBJudgg)
```

> 102. [accélérer le bpm](https://www.youtube.com/watch?v=BrO3GPy9mcY)

Reviewer: ```suggestion
102. [accelerating bpm](https://www.youtube.com/watch?v=BrO3GPy9mcY)
```

103. [bn promotion](https://www.youtube.com/watch?v=mxtAIr0RQQs)

> 104. [2 ans](https://www.youtube.com/watch?v=UIPp9hZ0TuM)

Reviewer: ```suggestion
104. [two years](https://www.youtube.com/watch?v=UIPp9hZ0TuM)
```

105. [lazer's hitsounds](https://www.youtube.com/watch?v=W0yUnAyiHnI)
106. [breakcore](https://www.youtube.com/watch?v=uE10Rbg_Fds)

> 107. [attentes](https://www.youtube.com/watch?v=qUpdmKDo7BQ)

Reviewer: ```suggestion
107. [expectations](https://www.youtube.com/watch?v=qUpdmKDo7BQ)
```

108. [spread rules](https://www.youtube.com/watch?v=b5l6ftgCcyM)
109. [cycles of change](https://www.youtube.com/watch?v=E0Ad9fN3d6A)

> 110. [ds origins](https://www.youtube.com/watch?v=JdAlJdAqeuo)

Reviewer: ```suggestion
110. [DS origins](https://www.youtube.com/watch?v=JdAlJdAqeuo)
```

111. [swing songs](https://www.youtube.com/watch?v=-qGgJC5LtIM)
112. [fanzhen](https://www.youtube.com/watch?v=f5p_mQu290g)
113. [developments before ppv2](https://www.youtube.com/watch?v=yp1jGc4o-Yw)

> 114. [2 ans de "loved"](https://www.youtube.com/watch?v=oNnDSdoTI2Y)

Reviewer: ```suggestion
114. [year 2 of "loved"](https://www.youtube.com/watch?v=oNnDSdoTI2Y)
```

115. [Mismagius & map difficiles](https://www.youtube.com/watch?v=TIr4kKyQjc0)
116. [Sotarks & not only pp](https://www.youtube.com/watch?v=p8A599ZYtZ8)
117. [anime](https://www.youtube.com/watch?v=pJRTpphiJzw)
118. [Mazzerin & metal](https://www.youtube.com/watch?v=FxmVFRYiuwE)
119. [KIRBY Mix](https://www.youtube.com/watch?v=asBB6Cf6l2k)

> 120. [ProfessionalBox & maps injouables](https://www.youtube.com/watch?v=pG0omZfzfpg)

Reviewer: ```suggestion
120. [ProfessionalBox & "unplayable" maps](https://www.youtube.com/watch?v=pG0omZfzfpg)
```

> 121. [évoluer des petite difficultés](https://www.youtube.com/watch?v=FAv5BsI3qLk)

Reviewer: ```suggestion
121. [evolving low difficulties](https://www.youtube.com/watch?v=FAv5BsI3qLk)
```

> 122. [apprendre taiko](https://www.youtube.com/watch?v=kM0bUdpptsY)

Reviewer: ```suggestion
122. [learning taiko](https://www.youtube.com/watch?v=kM0bUdpptsY)
```

> 123. [the BAT/QAT split](https://www.youtube.com/watch?v=zgKo04_xklY)

Reviewer: ```suggestion
123. [BAT/QAT split](https://www.youtube.com/watch?v=zgKo04_xklY)
```

> 124. [quelques pensées sur les mods](https://www.youtube.com/watch?v=58mUWF8U8HU)

Reviewer: ```suggestion
124. [some thoughts on mods](https://www.youtube.com/watch?v=58mUWF8U8HU)
```

Reviewer: Tu n'a pas appliquer cette modification

---
tags:
  - interview
  - pishi
  - pishifat
  - tutorial
  - video
  - videos
  - YouTube
  - vidéo
  - vidéos
  - tutoriel
---

# osu!mapping

*À ne pas confondre avec : [Beatmapping](/wiki/Beatmapping).*

> l'**osu!mapping** est une série de vidéos hebdomadaires créée par ![](/wiki/shared/flag/US.gif) **[pishifat](https://osu.ppy.sh/users/3178418)** a propos de l'écosystème du [mapping](/wiki/Beatmapping) et du [modding](/wiki/Modding). Ces vidéos peuvent être retrouvées sur [la chaîne YouTube de pishifat](https://www.youtube.com/user/Pishifat). La grande majorité des gens connais ces vidéos sous le nom de "pishifat videos" ou "pishi videos".

Reviewer 2: ```suggestion
l'**osu!mapping** est une série de vidéos hebdomadaires créée par ![](/wiki/shared/flag/US.gif) **[pishifat](https://osu.ppy.sh/users/3178418)** à propos de l'écosystème du [mapping](/wiki/Beatmapping) et du [modding](/wiki/Modding). Ces vidéos peuvent être retrouvées sur [la chaîne YouTube de pishifat](https://www.youtube.com/user/Pishifat). La grande majorité de la communauté connaît ces vidéos sous le nom de "pishifat videos" ou "pishi videos".
```

La série est devenue si réputée dans la communauté d'osu! qu'elle est devenue une ressource généralement recommandée aux nouveaux mappeurs. [L'équipe d'osu!](/wiki/People/The_Team) a récompensé pishifat du badge [Community Contributors](/wiki/People/Community_Contributors) pour sa "performance exemplaire dans la construction et l'éducation autour du domaine du mapping" en remerciement de ses efforts investis dans ce projet.

La série de vidéos hebdomadaires de l'osu!mapping s'arrêta en janvier 2019, mais il est toujours prévu d'ajouter de nouvelles vidéos dans le futur.

## Liens

- **[Chaîne YouTube](https://www.youtube.com/user/Pishifat)**
- [Patreon](https://www.patreon.com/pishifat) (actuellement fermé)
- [Twitter](https://twitter.com/pishifat)

## Histoire

> L'osu!mapping a débuté en tant que réponse à un [post reddit](https://www.reddit.com/r/osugame/comments/4upysl/request_for_mappers_of_this_sub/) demandant aux mappeurs de créer des vidéos commentées sur le mapping. Le 31 juillet 2016, pishifat posta sa première vidéo : [osu!mapping: concepts](https://www.youtube.com/watch?v=oL3az8FhlB4). Dans la vidéo, il explique comment il allait créer ces vidéos comme cela à été demandé sur Reddit, mais il finit par décider de faire autrement car un commentaire direct de mapping serait indigeste et n'expliquerait donc pas bien aux nouveaux mappeurs l'immense variété des concepts de mapping. À la place, il a choisi de commencer une série de vidéos dans laquelle chaque épisode se concentrerait sur un concept spécifique, expliquant son utilisation et son importance dans sa globalité sans laisser de flou.

Reviewer 2: ```suggestion
L'osu!mapping a débuté en tant que réponse à un [post reddit](https://www.reddit.com/r/osugame/comments/4upysl/request_for_mappers_of_this_sub/) demandant aux mappeurs de créer des vidéos commentées sur le mapping. Le 31 juillet 2016, pishifat posta sa première vidéo : [osu!mapping: concepts](https://www.youtube.com/watch?v=oL3az8FhlB4). Dans la vidéo, il explique comment il allait créer ces vidéos comme cela a été demandé sur Reddit, mais il finit par décider de faire autrement car un commentaire direct de mapping serait indigeste et n'expliquerait donc pas bien aux nouveaux mappeurs l'immense variété des concepts de mapping. À la place, il a choisi de commencer une série de vidéos dans laquelle chaque épisode se concentrerait sur un concept spécifique, expliquant son utilisation et son importance dans sa globalité sans laisser de flou.
```

---
tags:
  - interview
  - pishi
  - pishifat
  - tutorial
  - video
  - videos
  - YouTube
  - vidéo
  - vidéos
  - tutoriel
---

# osu!mapping

> *A ne pas confondre avec [Beatmapping](/wiki/Beatmapping).*

Reviewer 2: ```suggestion
*À ne pas confondre avec [Beatmapping](/wiki/Beatmapping).*
```

> *À ne pas confondre avec [Beatmapping](/wiki/Beatmapping).*

Reviewer: ```suggestion
*À ne pas confondre avec : [Beatmapping](/wiki/Beatmapping).*
```

> **osu!mapping** est une série de vidéo hebdomadaire créé par ![](/wiki/shared/flag/US.gif) **[pishifat](https://osu.ppy.sh/users/3178418)** a propos de l'écosystème du [mapping](/wiki/Beatmapping) et du [modding](/wiki/Modding). Ces vidéos peuvent être retrouvées sur [la chaîne youtube de pishifat](https://www.youtube.com/user/Pishifat). La grande majorité des gens connais ces vidéos sous le nom de "pishifat videos" ou "pishi videos".

Reviewer 2: ```suggestion
**osu!mapping** est une série de vidéos hebdomadaires créée par ![](/wiki/shared/flag/US.gif) **[pishifat](https://osu.ppy.sh/users/3178418)** a propos de l'écosystème du [mapping](/wiki/Beatmapping) et du [modding](/wiki/Modding). Ces vidéos peuvent être retrouvées sur [la chaîne youtube de pishifat](https://www.youtube.com/user/Pishifat). La grande majorité des gens connais ces vidéos sous le nom de "pishifat videos" ou "pishi videos".
```

> **osu!mapping** est une série de vidéos hebdomadaires créée par ![](/wiki/shared/flag/US.gif) **[pishifat](https://osu.ppy.sh/users/3178418)** a propos de l'écosystème du [mapping](/wiki/Beatmapping) et du [modding](/wiki/Modding). Ces vidéos peuvent être retrouvées sur [la chaîne youtube de pishifat](https://www.youtube.com/user/Pishifat). La grande majorité des gens connais ces vidéos sous le nom de "pishifat videos" ou "pishi videos".

Reviewer: ```suggestion
l'**osu!mapping** est une série de vidéos hebdomadaires créée par ![](/wiki/shared/flag/US.gif) **[pishifat](https://osu.ppy.sh/users/3178418)** a propos de l'écosystème du [mapping](/wiki/Beatmapping) et du [modding](/wiki/Modding). Ces vidéos peuvent être retrouvées sur [la chaîne YouTube de pishifat](https://www.youtube.com/user/Pishifat). La grande majorité des gens connais ces vidéos sous le nom de "pishifat videos" ou "pishi videos".
```

> La série est devenue si réputée dans la communauté d'osu! qu'elle est devenue une ressource généralement recommandée aux nouveaux mappeurs. [L'équipe d'osu!](/wiki/People/The_Team) a récompensé pishifat du badge [community contributors](/wiki/People/Community_Contributors) pour sa "performance exemplaire dans la construction et l'éducation autour du domaine du mapping" en remerciement de ses efforts investis dans ce projet.

Reviewer: ```suggestion
La série est devenue si réputée dans la communauté d'osu! qu'elle est devenue une ressource généralement recommandée aux nouveaux mappeurs. [L'équipe d'osu!](/wiki/People/The_Team) a récompensé pishifat du badge [Community Contributors](/wiki/People/Community_Contributors) pour sa "performance exemplaire dans la construction et l'éducation autour du domaine du mapping" en remerciement de ses efforts investis dans ce projet.
```

> La série est devenue tellement réputée dans la communauté osu! que c'est une ressource qui est généralement recommandé aux nouveaux mappeurs. [L'équipe d'osu!](/wiki/People/The_Team) à récompensé pishifat du badge [community contributors](/wiki/People/Community_Contributors) pour sa "performance exemplaire dans la construction et l'éducation autour du domaine du mapping" et pour remercier tout ses efforts investi dans ce projet.

Reviewer 2: ```suggestion
La série est devenue si réputée dans la communauté d'osu! qu'elle est devenue une ressource généralement recommandée aux nouveaux mappeurs. [L'équipe d'osu!](/wiki/People/The_Team) a récompensé pishifat du badge [community contributors](/wiki/People/Community_Contributors) pour sa "performance exemplaire dans la construction et l'éducation autour du domaine du mapping" en remerciement de ses efforts investis dans ce projet.
```

> La suite des vidéos hebdomadaire de l'osu!mapping s'arrêta en janvier 2019, mais il y a toujours des vidéos prévus qui compte être ajoutés dans la série.

Reviewer 2: ```suggestion
La série de vidéos hebdomadaires de l'osu!mapping s'arrêta en janvier 2019, mais il est toujours prévu d'ajouter de nouvelles vidéos dans le futur.
```

## Liens

- **[Chaîne YouTube](https://www.youtube.com/user/Pishifat)**
- [Patreon](https://www.patreon.com/pishifat) (actuellement fermé)
- [Twitter](https://twitter.com/pishifat)

## Histoire

> l'osu!mapping a débuté comme une réponse a un [post reddit](https://www.reddit.com/r/osugame/comments/4upysl/request_for_mappers_of_this_sub/) demandant au mappeurs de créer des vidéos commentés sur le mapping. Le 31 juillet 2016, pishifat posta sa première vidéo : [osu!mapping: concepts](https://www.youtube.com/watch?v=oL3az8FhlB4). Dans la vidéo, il explique comment il va faire ces vidéos comme cela à été demander sur reddit, mais il finit par décider de faire autrement car un commentaire direct de mapping serait indigeste et n'expliquerait donc pas bien aux nouveaux mappeurs l'immense variété des concept de mapping. A la place, il a choisi de commencer une série de vidéos ou chaque épisode se concentrerais sur un concept spécifique, expliquant son utilisation et importance dans sa globalité sans laisser de floues.

Reviewer 2: ```suggestion
L'osu!mapping a débuté en tant que réponse à un [post reddit](https://www.reddit.com/r/osugame/comments/4upysl/request_for_mappers_of_this_sub/) demandant aux mappeurs de créer des vidéos commentées sur le mapping. Le 31 juillet 2016, pishifat posta sa première vidéo : [osu!mapping: concepts](https://www.youtube.com/watch?v=oL3az8FhlB4). Dans la vidéo, il explique comment il allait créer ces vidéos comme cela à été demandé sur Reddit, mais il finit par décider de faire autrement car un commentaire direct de mapping serait indigeste et n'expliquerait donc pas bien aux nouveaux mappeurs l'immense variété des concepts de mapping. À la place, il a choisi de commencer une série de vidéos dans laquelle chaque épisode se concentrerait sur un concept spécifique, expliquant son utilisation et son importance dans sa globalité sans laisser de flou.
```

> <!-- TODO: this is not really concrete history -->

Reviewer 3: ```suggestion
```

> La série s'est concentré  au début sur le coté tutoriel, mais a commencé a se diversifier après quelques mois pour inclure des discussions a propos de l'état du mapping et du modding, des commentaires personnels et des opinions, et des interviews. Il y a a aussi eu des projets de recherches présentés sous la forme de vidéos d'osu!mapping, notamment [une étude des influences des mappeurs](https://www.youtube.com/watch?v=TigwDc5iOcw).

Reviewer 2: ```suggestion
La série s'est concentrée au début sur le côté tutoriel, mais a commencé à se diversifier après quelques mois pour inclure des discussions à propos de l'état du mapping et du modding, des commentaires personnels et des opinions, et des interviews. Il y a aussi eu des projets de recherche présentés sous la forme de vidéos d'osu!mapping, notamment [une étude sur l'influence des mappeurs](https://www.youtube.com/watch?v=TigwDc5iOcw).
```

> La série s'est concentrée au début sur le côté tutoriel, mais a commencé à se diversifier après quelques mois pour inclure des discussions à propos de l'état du mapping et du modding, des commentaires personnels et des opinions, et des interviews. Il y a aussi eu des projets de recherche présentés sous la forme de vidéos d'osu!mapping, notamment [une étude sur l'influence des mappeurs](https://www.youtube.com/watch?v=TigwDc5iOcw).

Reviewer: ```suggestion

La série s'est concentrée au début sur le côté tutoriel, mais a commencé à se diversifier après quelques mois pour inclure des discussions à propos de l'état du mapping et du modding, des commentaires personnels et des opinions, et des interviews. Il y a aussi eu des projets de recherche présentés sous la forme de vidéos d'osu!mapping, notamment [une étude sur l'influence des mappeurs](https://www.youtube.com/watch?v=TigwDc5iOcw).
```
